diff --git a/src/opts/SkRasterPipeline_opts.h b/src/opts/SkRasterPipeline_opts.h
index 06ccfd7df26b..64f4ea7ad57b 100644
--- a/src/opts/SkRasterPipeline_opts.h
+++ b/src/opts/SkRasterPipeline_opts.h
@@ -1206,7 +1206,7 @@ static void start_pipeline(size_t dx, size_t dy,
     }
 }
 
-#if SK_HAS_MUSTTAIL
+#if SK_HAS_MUSTTAIL && !defined(__powerpc64__)
     #define JUMPER_MUSTTAIL [[clang::musttail]]
 #else
     #define JUMPER_MUSTTAIL
