diff --git a/src/core/SkRasterPipeline.h b/src/core/SkRasterPipeline.h
index dc5fb4b8f249..2b285a8ff360 100644
--- a/src/core/SkRasterPipeline.h
+++ b/src/core/SkRasterPipeline.h
@@ -24,7 +24,8 @@ enum SkColorType : int;
 struct SkImageInfo;
 struct skcms_TransferFunction;
 
-#if __has_cpp_attribute(clang::musttail) && !defined(__EMSCRIPTEN__) && !defined(SK_CPU_ARM32)
+#if __has_cpp_attribute(clang::musttail) && !defined(__EMSCRIPTEN__) && \
+        !defined(SK_CPU_ARM32) && !defined(SK_CPU_PPC)
     #define SK_HAS_MUSTTAIL 1
 #else
     #define SK_HAS_MUSTTAIL 0
